.changeset/curly-bikes-protect.md

```diff
@@ -0,0 +1,5 @@
+---
+'@itwin/itwinui-react': minor
+---
+
+`Stepper` now allows users to pass in custom icon or content in each step circle to indicate step completion.
```

Comment:
```suggestion
`Stepper` now allows passing custom icon or content in each step circle to indicate step completion. This can be done by using a `stepContent` property in each item of the `steps` array.
```

Reply:
I have made the change.